helpers/lib_tasks_docker_release.py

```diff
@@ -62,6 +62,207 @@ def _get_dev_version(version: str, container_dir_name: str) -> str:
+    :param superslow_tests: run superslow tests
+    :param qa_tests: run QA tests
+    """
+    if skip_tests:
+        _LOG.warning("Skipping all tests")
+        return
+    if fast_tests:
+        hlitapyt.run_fast_tests(ctx, stage=stage, version=version)
+    if slow_tests:
+        hlitapyt.run_slow_tests(ctx, stage=stage, version=version)
+    if superslow_tests:
+        hlitapyt.run_superslow_tests(ctx, stage=stage, version=version)
+    if qa_tests:
+        hlitapyt.run_qa_tests(ctx, stage=stage, version=version)
+
+
+def _docker_tag_and_push_multi_arch_image(
+    ctx: Any,
+    version: str,
+    base_image: str,
+    target_registry: str,
+    container_dir_name: str,
+    stage: str,
+    target_stage: str,
+) -> None:
+    """
+    Tag and push a multi-arch image to the target registry using `docker buildx
+    imagetools`.
+
+    :param ctx: invoke context
+    :param version: version to tag the image with
+    :param base_image: base name of the image
+    :param target_registry: target Docker registry to push to (e.g.,
+        "aws_ecr.ck" or "dockerhub.causify")
+    :param container_dir_name: directory where Dockerfile is located
+    :param stage: source stage of the image ("local" or "prod")
+    :param target_stage: target stage to push the image as ("dev" or
+        "prod")
+    """
+    hlitadoc.docker_login(ctx, target_registry)
+    # Get version string.
+    if stage == "local":
+        image_stage_version = _get_dev_version(version, container_dir_name)
+    else:
+        image_stage_version = hlitadoc.resolve_version_value(
+            version, container_dir_name=container_dir_name
+        )
+    image_versioned = hlitadoc.get_image(base_image, stage, image_stage_version)
+    _LOG.info(
+        "Pushing the %s image %s to the target_registry %s ",
+        stage,
+        image_versioned,
+        target_registry,
+    )
+    if target_registry == "aws_ecr.ck":
+        # Use AWS Docker registry.
+        target_base_image = ""
+    elif target_registry == "dockerhub.causify":
+        # Use public GitHub Docker registry.
+        base_image_name = hrecouti.get_repo_config().get_docker_base_image_name()
+        target_base_image = f"causify/{base_image_name}"
+    else:
+        raise ValueError(
+            f"Invalid target Docker image registry='{target_registry}'"
+        )
+    # Tag and push the base image as versioned target image
+    target_versioned_image = hlitadoc.get_image(
+        target_base_image, target_stage, image_stage_version
+    )
+    cmd = f"docker buildx imagetools create -t {target_versioned_image} {image_versioned}"
+    hlitauti.run(ctx, cmd)
+    # Tag and push the base image as target image.
+    latest_version = None
+    latest_image = hlitadoc.get_image(
+        target_base_image, target_stage, version=latest_version
+    )
+    cmd = f"docker buildx imagetools create -t {latest_image} {image_versioned}"
+    hlitauti.run(ctx, cmd)
+
+
+def _rollback_image(
```

Comment:
Let's not make another function, but just move all the code inside the `_docker_rollback_image()`.

So, in this way, for example, the `def docker_rollback_dev_image()` will call the `_docker_rollback_image()`.